DiscordChatExporter.Core/Exporting/Writers/MarkdownVisitors/HtmlMarkdownVisitor.cs

```diff
@@ -123,7 +123,7 @@ protected override MarkdownNode VisitEmoji(EmojiNode emoji)
         var jumboClass = _isJumbo ? "chatlog__emoji--large" : "";
 
         _buffer
-            .Append($"<img loading=\"lazy\" class=\"chatlog__emoji {jumboClass}\" alt=\"{emoji.Name}\" title=\"{emoji.Code}\" src=\"{emojiImageUrl}\">");
+            .Append($"<img loading=\"lazy\" class=\"chatlog__emoji {jumboClass}\" alt=\"{emoji.Name}\" title=\"{emoji.Code}\" src=\"{_context.ResolveMediaUrlAsync(emojiImageUrl)}\">");
```

Comment:
This needs to be awaited but we can't do it here

Reply:
Hmm that makes it quite more difficult then since the message content is place into the HTML context using the @Raw(...) function. Is there an example I can maybe check out to resolve this in the best way?

Reply:
Unfortunately no, I think it will take a bit of creativity to come up with a good way to solve this